examples/jsm/postprocessing/SSAARenderPass.js

```diff
@@ -95,7 +95,31 @@ class SSAARenderPass extends Pass {
 		const roundingRange = 1 / 32;
 		this.copyUniforms[ 'tDiffuse' ].value = this.sampleRenderTarget.texture;
 
-		const width = readBuffer.width, height = readBuffer.height;
+		let viewOffset = {
+			
+			fullWidth: readBuffer.width,
+			
+			fullHeight: readBuffer.height,
+			
+			offsetX: 0,
+			
+			offsetY: 0,
+			
+			width: readBuffer.width,
+			
+			height: readBuffer.height
+		
+		};
+
+		let originalViewOffset = null;
+
+		if ( this.camera.setViewOffset ) {
```

Comment:
It would be good to avoid a check for `setViewOffset()`. Wouldn't the code also work if you just remove this `if` statement and always execute the next two lines?

Reply:
@Mugen87 

The aim was to respect the existing `setViewOffset` and `clearViewOffset` conditionals.

Removing it would assume a `view` property always exists, and if it didn't an error would occur as `undefined` !== `null`.

Checking for the existence of `setViewOffset` proves that a `view` property exists and will behave as `null` or `Object`.

If all cameras provide a `view` property with the same behavior and methods, then all the conditionals in question can go.

Reply:
My aim was to remain consistent with the existing `setViewOffset` and `clearViewOffset` conditionals.

But, by virtue of JavaScript those lines wouldn't cause an issue as `Object.assign( {}, undefined )` is `{}`.

So, I'll drop the conditional.